src/style/themes/base/base-theme.config.js

```diff
@@ -273,10 +273,15 @@ export default (palette) => {
     },
 
     search: {
-      active: palette.goldTint(50),
+      active: "#FFB500",
```

Comment:
@DipperTheDan can you check with @harpalsingh that we want to change the `border-bottom-color` on `:hover` to be `#FFB500` which is the same as the as the `Textbox` focus outline. Or if it should remain as `goldTint(50)`.

Thanks!